dashbuilder/dashbuilder-client/dashbuilder-renderers/dashbuilder-renderer-lienzo/pom.xml

```diff
@@ -18,7 +18,7 @@
   <dependencies>
 
     <dependency>
-      <groupId>com.ahome-it</groupId>
+      <groupId>org.kie</groupId>
```

Comment:
There are some issues with this:

1. After this change is done, then we have to exclude `com.ahome-it:lienzo-core` from the `lienzo-charts` dependency below, otherwise there are duplicated classes.

2. Even if it is excluded, there is a compilation error when building `dashbuilder-app`:
```
Compiling module org.dashbuilder.DashbuilderShowcase
[INFO]    Tracing compile failure path for type 'com.ait.lienzo.charts.client.core.json.validators.PieChartDataValidator'
[INFO]       [ERROR] Errors in 'jar:file:/home/mmacik/.m2/repository/com/ahome-it/lienzo-charts/1.0.241-RC1/lienzo-charts-1.0.241-RC1.jar!/com/ait/lienzo/charts/client/core/json/validators/PieChartDataValidator.java'
[INFO]          [ERROR] Line 34: The type new IAttributeTypeValidator(){} must implement the inherited abstract method IAttributeTypeValidator.getTypeName()
[INFO]          [ERROR] Line 34: The type new IAttributeTypeValidator(){} must implement the inherited abstract method IAttributeTypeValidator.isIgnored()
[INFO]    [ERROR] Aborting compile due to errors in some input files
```
The root cause is that the lienzo-charts uses an interface from lienzo-core which looks like [this](https://github.com/dikalo/lienzo-core/blob/efbf4c72694004d553f8e967f1e4a87f51438891/src/main/java/com/ait/lienzo/client/core/shape/json/validators/IAttributeTypeValidator.java#L21). However, the `org.kie:lienzo-core` has this interface slightly changed to [this](https://github.com/kiegroup/lienzo-core/blob/master/src/main/java/com/ait/lienzo/client/core/shape/json/validators/IAttributeTypeValidator.java#L21), so obviously there are methods which are not implemented. @romartin is this ok or should we update it appropriately?

Reply:
@romartin ^^

@MarianMacik yes, I started an email thread on this due mismatched of lienzo versions